index.html

```diff
@@ -7,6 +7,6 @@
     <title>Git Basics</title>
 </head>
 <body>
-    <h3>Moe - This session is awesome</h3>
+    <h3>Moe - This session is awesome</h3><h3>Suzzie - I am here</h3>
```

Comment:
Hey Suzzie, please put that h3 in the line below

Reply:
I will fix in VS Code next time Thank you